src/io/internal/pipelines/dicom/apply-pstate-to-image.cxx

```diff
@@ -80,32 +84,49 @@ static OFLogger dcmp2pgmLogger = OFLog::getLogger("dcmtk.apps." OFFIS_CONSOLE_AP
 static char rcsid[] = "$dcmtk: " OFFIS_CONSOLE_APPLICATION " v"
```

Comment:
Consider updating the application name associated with `OFFIS_CONSOLE_APPLICATION` above.

Similarly:
* `rcsid` value should be updated.
* variable `dcmp2pgmLogger` renamed

Reply:
@jcfr any suggestions for rcsid modification? should we append "itk-wasm" to it at the end?